src/simulate/assets/spaces/box.py

```diff
@@ -0,0 +1,155 @@
+import warnings
```

Comment:
We should copy OpenAI MIT licence at the top of each of their files.

Other than that, looks good to me!